### DISCUSSION TITLE
ENOPKG Missing `package.json` file

### DISCUSSION THREAD

**Discussion by kilosonc** [semantic-release/semantic-release#2014] [Configuration]
## Current behavior

When I use semantic release on this [project](https://github.com/casbin/redis-watcher), I encountered this error. I'm not a js programmer, so it's not easy for me to solve this question. I ask for help.

## Environment

plz refer to https://github.com/casbin/redis-watcher
- **semantic-release** version: 17.2.4
- CI environment: github action
- Plugins used:     "@semantic-release/commit-analyzer", "@semantic-release/release-notes-generator", "@semantic-release/github"
- **semantic-release** configuration: https://github.com/casbin/redis-watcher/blob/master/.github/workflows/ci.yml https://github.com/casbin/redis-watcher/blob/master/releaserc.json
- CI logs: https://github.com/casbin/redis-watcher/runs/2991949805

**Comment by travi** [comment]
it appears that semantic-release is not picking up your config because the file is not preceded with a `.`. the valid config file names are [outlined in our docs](https://semantic-release.gitbook.io/semantic-release/usage/configuration#configuration-file)
